types/applepayjs/index.d.ts

```diff
@@ -143,6 +148,12 @@ declare class ApplePaySession extends EventTarget {
      */
     completeShippingMethodSelection(update: ApplePayJS.ApplePayShippingMethodUpdate): void;
 
+    /**
+     * Completes the selection of a shipping method with an update for Apple Pay JS version 3.
+     * @param update - The updated shipping method.
```

Comment:
```suggestion
     * Completes the entry of a coupon code with an update.
     * @param update - The updated coupon code.
```

Reply:
Done, thanks!
Could you please take a look?